hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerImpl.java

```diff
@@ -235,21 +241,21 @@ public static void cleanup() throws Exception {
     FileUtils.deleteDirectory(dir);
   }
 
+  @BeforeEach
+  public void init() throws Exception {
+    createBucket(VOLUME_NAME, BUCKET_NAME, false);
+    createBucket(VOLUME_NAME, BUCKET2_NAME, false);
+    createBucket(VOLUME_NAME, VERSIONED_BUCKET_NAME, true);
+  }
+
   @AfterEach
   public void cleanupTest() throws IOException {
     mockContainerClient();
-    List<OzoneFileStatus> fileStatuses = keyManager
-        .listStatus(createBuilder().setKeyName("").build(), true, "", 100000);
-    for (OzoneFileStatus fileStatus : fileStatuses) {
-      if (fileStatus.isFile()) {
-        writeClient.deleteKey(
-            createKeyArgs(fileStatus.getKeyInfo().getKeyName()));
-      } else {
-        writeClient.deleteKey(createKeyArgs(OzoneFSUtils
-            .addTrailingSlashIfNeeded(
-                fileStatus.getKeyInfo().getKeyName())));
-      }
-    }
+    org.apache.hadoop.fs.Path volumePath = new org.apache.hadoop.fs.Path(OZONE_URI_DELIMITER, VOLUME_NAME);
```

Comment:
import `org.apache.hadoop.fs.Path`

Reply:
There is already import of `import java.nio.file.Path;` so it's a conflict.